src/main/webapp/admin_dashboard.html

```diff
@@ -1,127 +1,93 @@
-			<div id="popup">
-				<h2>Edit Task</h2>
-				<form action="/admin-user-tasks" method="POST">
-					Task Details: <input type="text" id="task-detail-input" name="task-detail-input">
-					Reward: <input type="text" id="reward-input" name="reward-input" min="0" max="200">
-					<select name="edit-category-input" id="edit-category-input">
-						<option value="garden">Garden</option>
-						<option value="shopping">Shopping</option>
-						<option value="pets">Pets</option>
-						<option value="misc">Misc</option>
-					</select>
-                    <input type="submit" value="Submit">
-					<input type="hidden" name="task-id" id="task-id-input">
-				</form>
-				<a href="#" id="popup-close">&times;</a>
-                
-			</div>
-		</div>
-	</body>
-</html>
+<!DOCTYPE html>
+<!--
+	Copyright 2020 Google LLC
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+-->
+
+<html>
+	<head>
+		<title>Admin Dashboard</title>
+		<link rel="stylesheet" href="admin_dashboard_style.css">
+		<script src='https://kit.fontawesome.com/a076d05399.js'></script>
+		<script src="https://www.gstatic.com/charts/loader.js"></script>
+		<script type="text/javascript" src="config.js"></script>
+		<script src="admin_dashboard_script.js"></script>
+	</head>
+	<body>
+		<div class="wrapper">
+			<main>
+				<h1><a href="/"><i class="fas fa-user-circle fa-2x" title="Go to User Page" aria-hidden="true"></i></a> Admin Dashboard</h1>
+				<div class="flex-grid">
+					<div class="card-container">
+						<h2>Administrator Tasks<a href="#admin-popup"><span><i class="fas fa-plus fa-2x"></i></span></a></h2>
+						<div id="card-div">
+							<ul id="admin-tasks">
+							</ul>
+						</div>
+					</div>
+					<div id="chart-container">
+						<h2>Usage Statistics</h2>
+						<div id="chart-div"></div>
+					</div>
+				</div>
+				<div class="flex-grid">
+					<div id="map-container">
+						<h2>Map</h2>
+						<div id="map-div"></div>
+					</div>
+					<div class="card-container">
+						<h2> <span>Last 10 User Tasks</span></h2>
+						<div id="card-div">
+							<ul id="user-tasks">
+							</ul>
+						</div>
+					</div>
+				</div>
+			</main>
+		</div>
+		<div class="popup-overlay" id="user-popup">
+			<div id="popup">
+				<h2>Edit Task</h2>
+				<form action="/admin-user-tasks" method="POST">
+					Task Details: <input type="text" id="task-detail-input" name="task-detail-input">
+					Reward: <input type="text" id="reward-input" name="reward-input" min="0" max="200">

```

Comment:
I would suggest adding a onsubmit function to check whether the input task info field is empty or not. We add this new feature to the form submission process on the user profile page recently and it could prevent the users from submitting empty task info or task overview.